### ISSUE TITLE
v5.0.0 introduced lot of stuff for ObjectStorage, but documentation doesn't even mention them

### ISSUE THREAD

**Issue by jylitalo** [UpCloudLtd/terraform-provider-upcloud#498]
New stuff:
```
upcloud_managed_object_storage: upcloud_managed_storage_policy resource for setting up policies
upcloud_managed_object_storage: upcloud_managed_storage_policies data source for policies
upcloud_managed_object_storage: upcloud_managed_storage_user resource for user management
upcloud_managed_object_storage: upcloud_managed_storage_user_policy resource for attaching policies to users
upcloud_managed_object_storage: iam_url property to upcloud_managed_storage.endpoint
upcloud_managed_object_storage: sts_url property to upcloud_managed_storage.endpoint
upcloud_managed_object_storage: required status property to upcloud_managed_storage_user_access_key
```
but we have nothing about them in documentation:
<img width="313" alt="Screenshot 2024-03-01 at 14 10 07" src="https://github.com/UpCloudLtd/terraform-provider-upcloud/assets/765664/532075c0-316b-4537-870d-b1994959f184">

so right now only documentation is at: https://github.com/UpCloudLtd/terraform-provider-upcloud/tree/main/internal/service/managedobjectstorage

**Comment by jylitalo** [comment]
Purely out of luck found them from different path. So instead of `upcloud provider` -> `Object Storage` -> `Resources`, they are directly under `upcloud provider` -> `Resources`
<img width="313" alt="Screenshot 2024-03-01 at 14 16 28" src="https://github.com/UpCloudLtd/terraform-provider-upcloud/assets/765664/1ac779ff-3a8b-47d6-a974-31eff63ac674">

**Comment by kangasta** [comment]
Hello, thanks for reporting this! As you mentioned, those are currently under _Resources_ group in the provider documentation, because no group was defined for them. We'll fix the grouping logic to move those under the correct group.

**Comment by kangasta** [comment]
This is now fixed with v5.0.1
